### ISSUE TITLE
LCHARG is wrongly handled with `retain_charge_density`

### ISSUE THREAD

**Issue by dhoulek** [pyiron/pyiron_atomistics#1513]
**Summary**

The keys `retain_charge_density` in `calc_minimize`, `calc_static` and `calc_md` in vasp/base.py wrongly handle the VASP tag. The tag is written only when `retain_charge_density=True`; otherwise, no tag is written to INCAR. In that case, a default value is used, which is, however, TRUE. Therefore, the CHGCAR is always written.

**pyiron Version and Platform**

0.5.3 Linux, but should also apply to the current stable version on github

**Expected Behavior**

For `retain_charge_density=False`, a tag `LCHARG=.FALSE.` is written to INCAR.
For `retain_charge_density=True`, no tag is written and default behavior is invoked (CHGCAR is written). Or, LCHARG tag is explicitly written with `.TRUE.` value.

**Actual Behavior**

For `retain_charge_density=False`, no `LCHARG` tag is written to INCAR, and hence default behavior (`LCHARG=.TRUE.`) is invoked.

**Further Information, Files, and Links**
in vasp/base.py:
lines 1069-1070
lines 1103-1104
lines 1152-1153

**Comment by pmrv** [comment]
Thanks for the report!

I'm working a on fix in #1514, but in the mean time you can use
```
job.input.incar['LCHARG'] = '.FALSE.'
```

**Comment by jan-janssen** [comment]
@dhoulek The new version is now available on conda https://anaconda.org/conda-forge/pyiron_atomistics you can install it using: 
```
conda install -c conda-forge pyiron_atomistics=0.6.10
```
